app/controllers/concerns/price_group_members_controller.rb

```diff
@@ -56,4 +57,8 @@ def load_price_group_and_ability!
     @price_group = current_facility.price_groups.find(params[:price_group_id])
     @price_group_ability = Ability.new(current_user, @price_group, self)
   end
+
+  def authorize_price_group_member
+    @price_group_ability.authorize! :create, PriceGroupMember
```

Comment:
While it's likely fine in this scenario, you're calling this before filter on `destroy` as well, which is technically a different permission. Could you use `action_name`?